pkg/gui/gui.go

```diff
@@ -449,7 +496,7 @@ func (gui *Gui) layout(g *gocui.Gui) error {
 		}
 	}
 
-	if v, err := g.SetView("information", optionsVersionBoundary-1, optionsTop, width, optionsTop+2, 0); err != nil {
+	if v, err := g.SetView("information", optionsVersionBoundary-1, height+2, width, height, 0); err != nil {
```

Comment:
```suggestion
	if v, err := g.SetView("information", optionsVersionBoundary-1, height-2, width, height, 0); err != nil {
```
Currently the information and appStatus views are no visible